Paper-R-Us is a national distributor of printer and copier paper for commercial use. The data file called
Sales contains the annual, year-to-date sales values for each of the company’s customers. Suppose the
internal audit department has decided to audit a sample of these accounts. Specifically, they have
decided to sample 36 accounts. However, before they actually conduct the in-depth audit (a process
that involves tracking all transactions for each sampled account), they want to be sure that the sample
they have selected is representative of the population.

a) Compute the population mean.

In [1]:
import pandas as pd
import numpy as np

In [2]:
sales = pd.read_excel('D://Business Statistic/MANB1123-master/MANB1123-master/Data_Set/Sales.xlsx')
sales.head()

Account #  Sales Region
0      39004  10396   East
1      94574   9737   West
2      24608   9215   West
3      32790  10331  South
4      23208  11580   East

In [3]:
sales_value = sales['Sales']
sales_value.head()

0    10396
1     9737
2     9215
3    10331
4    11580
Name: Sales, dtype: int64

In [4]:
sales_value.mean() #mean value for sales population

9987.436

b) Use all the data in the population to develop a frequency distribution and histogram.

In [9]:
import plotly.plotly as py
import plotly.graph_objs as go

In [13]:
x = sales['Sales']
data2 = [go.Histogram(x=x)]

py.iplot(data2, filename='basic histogram')

In [36]:
highest = sales_value.max() #highest value
lowest = sales_value.min()   #lowest value
bar = 5
width = (highest - lowest)/ bar #bin size

In [39]:
trace1 = go.Histogram(
    x=sales['Sales'],
    xbins=dict(start=6000, end = 13000, size = width),
    opacity=0.75,
    name='column 0'
)

data = [trace1]

layout = go.Layout(
    title='Paper-R-Us Sales',
    barmode='overlay',
    xaxis=dict(
    title='Sales Value'
    ),
    yaxis=dict(
        title='Frequency'
    )


) 
fig = go.Figure(data=data, layout=layout)
py.iplot(fig)

c) Calculate the proportion of accounts for customers in each region of the country.

In [19]:
sales.groupby('Region').size()

Region
East       99
Midwest    44
South      79
West       28
dtype: int64

In [21]:
#total population
sales.shape[0]

250

In [24]:
#East proportion
east = 99 / 250
print (east)

#Midwest proportion
midwest = 44/250
print (midwest)

#South proportion
south = 79 / 250
print (south)

#west proprtion
west = 28 /250
print (west)

0.396
0.176
0.316
0.112


d) Select a random sample of accounts. Develop a frequency distribution for these sample data.
Compare this distribution to that of the population. (Hint: You might want to consider using
relative frequencies for comparison purposes.)

In [43]:
#random sample
early = sales.head(20) #first 20 of sales value
early

Account #  Sales   Region
0       39004  10396     East
1       94574   9737     West
2       24608   9215     West
3       32790  10331    South
4       23208  11580     East
5       59515   9762  Midwest
6       47612   8239     East
7       51835  10195    South
8       41108   8818  Midwest
9       82218   8954  Midwest
10      77863  10420    South
11      66235   9774    South
12      74948   9229     East
13      58184  10164    South
14      19974   9469     East
15      61932  11090  Midwest
16      20191  10080     East
17      35100  10685  Midwest
18      21758   9885     East
19      34926   9220     East

In [47]:
#create frequency distribution
trace1 = go.Histogram(
    x=early['Sales'],
    xbins=dict(start=8000, end = 13000, size = 1000),
    opacity=0.75,
    name='column 0'
)

data = [trace1]

layout = go.Layout(
    title='Paper-R-Us Sales',
    barmode='overlay',
    xaxis=dict(
    title='Sales Value'
    ),
    yaxis=dict(
        title='Frequency'
    )


) 
fig = go.Figure(data=data, layout=layout)
py.iplot(fig)

From the distribution, it can be seen the data is normally distribute and looks almost similar to the population distribution

e) Construct a 95% confidence interval estimate for the population mean sales per customer.
Discuss how you would use this interval estimate to help determine whether the sample is a
good representation of the population. (Hint: You may want to use the finite population
correction factor since the sample is large relative to the size of the population.)

In [49]:
sales['Sales'].describe()

count      250.000000
mean      9987.436000
std        925.733395
min       6356.000000
25%       9390.000000
50%       9958.000000
75%      10588.000000
max      12312.000000
Name: Sales, dtype: float64

In [50]:
import scipy.stats as st 
import math as m

In [148]:
#Get Z-score
a = 0.95 /2

#use z-table(mean);P(0 < Z < z)
z = 0.1808 # a = 0.475
print (z)

0.1808


In [149]:
std = 925.733395
n = 250

s = m.sqrt(n) 

e = z * (std / s)
print (e)

10.585572539957674


the margin of error is +/- 10.59

f)Use the information developed in parts a–e to draw a conclusion about whether the sample is a
representative sample of the population. What other information would be desirable? Discuss.

Any changes in confidence level will affect margin of error. Increase in sample size will decrease margin of error and standard error